src/index.jsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
+  <BrowserRouter>
```

Comment:
You shouldn't have changed this

Reply:
Why? how will routes work with StrictMode

Reply:
React.Scrictmode won't affect the routes from working